scalding-spark/src/main/scala/com/twitter/scalding/spark_backend/SparkWriter.scala

```diff
@@ -213,6 +213,6 @@ class SparkWriter(val sparkMode: SparkMode) extends Writer {
       (nextState.copy(id = nextState.id + 1), (nextState.id, acts))
     }
     // now we run the actions:
-    Future.traverse(acts) { fn => fn() }.map(_ => (id, ExecutionCounters.empty))
+    CFuture(Future.traverse(acts) { fn => fn() }.map(_ => (id, ExecutionCounters.empty)), CancellationHandler.empty)
```

Comment:
I think we want to use `CFuture.uncancellable` or something here.

Reply:
👍